app/validation/error_messages.py

```diff
@@ -22,4 +22,8 @@
     'MAX_LENGTH_EXCEEDED': 'Your answer is too long, it has to be less than %(max)d characters.',
     'INVALID_DATE': 'Enter a valid date.',
     'INVALID_DATE_RANGE': "Enter a 'period to' date later than the 'period from' date.",
+    'DATE_PERIOD_TOO_SMALL': 'Enter a reporting period greater than %(min)s.',
+    'DATE_PERIOD_TOO_LARGE': 'Enter a reporting period less than %(max)s.',
```

Comment:
The error messages are wrong.
```
"period_limits": {
	"minimum": {
		"days": 31
     },
	"maximum": {
		"days": 31
	}
}
```
If you use the above settings and if the period difference is 32, the error message shown is:
`1) Enter a reporting period less than 31 days.`
It should be: `less than 32 days` or `less than or equal to 31 days`

If the period difference is 30, the error message shown is:
`1) Enter a reporting period greater than 31 days.`
It should be: `greater than 30 days` or `greater than or equal to 31 days`

Reply:
Schema validator wont allow same period limits but error messages are a bit strange there i will dig around a little and get back

Reply:
Just looking at these again the error messages are correct

Reply:
Minimum and maximum are generally inclusive. But also your point about schema validator not allowing?. For me, it says it is a valid schema?